From 97b2a61d74edebad43ad09612c92a0341090f165 Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@cryptomilk.org>
Date: Tue, 25 Sep 2018 14:35:43 +0200
Subject: [PATCH] config: Fix building without globbing support

Signed-off-by: Andreas Schneider <asn@cryptomilk.org>
(cherry picked from commit f709c3ac585f7b47317758b8693a6d104b30f951)
Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
Upstream status: commit 97b2a61d74 (stable-0.8 branch)

 src/config.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/config.c b/src/config.c
index df6b48bf6d5e..3d87a1780a58 100644
--- a/src/config.c
+++ b/src/config.c
@@ -462,7 +462,7 @@ static int ssh_config_parse_line(ssh_session session, const char *line,
 
       p = ssh_config_get_str_tok(&s, NULL);
       if (p && *parsing) {
-#ifdef HAVE_GLOB
+#if defined(HAVE_GLOB) && defined(HAVE_GLOB_GL_FLAGS_MEMBER)
         local_parse_glob(session, p, parsing, seen);
 #else
         local_parse_file(session, p, parsing, seen);
-- 
2.19.1

